groups/ecs/data.tf

```diff
@@ -0,0 +1,75 @@
+data "vault_generic_secret" "secrets" {
+  path = "team-${var.team}/${var.account_name}/${var.region}/${var.environment}/${var.repository_name}"
+}
```

Comment:
This doesn't follow our tytpical namespacing on Vaultr secrets and we don't store secrets based on the source repository name.

Reply:
Which path should be used, please?